### PR TITLE
algo package update for visible on hit feature

### PR DESCRIPTION
**Pull request by avsek477**
Depends on: 

- https://github.com/bitfinexcom/bfx-api-node-core/pull/29
- https://github.com/bitfinexcom/bfx-hf-algo/pull/194

Package update for hf-algo and api-node-core.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,10 +19,10 @@
   ],
   "dependencies": {
     "aes-js": "^3.1.2",
-    "bfx-api-node-core": "^1.5.5",
+    "bfx-api-node-core": "git+https://github.com/avsek477/bfx-api-node-core.git#orders-models-update",
```
Comment (avsek477) [inline package.json:22]: needs to be changed after this [PR](https://github.com/bitfinexcom/bfx-api-node-core/pull/29) is merged.

File: package.json [hunk 2]
```diff
@@ -19,10 +19,10 @@
...
     "bfx-api-node-plugin-wd": "^1.0.4",
     "bfx-api-node-rest": "^4.3.1",
-    "bfx-hf-algo": "git+https://github.com/bitfinexcom/bfx-hf-algo.git#v4.7.3",
+    "bfx-hf-algo": "git+https://github.com/avsek477/bfx-hf-algo.git#add-visible-on-hit-for-algo-order",
```
Comment (avsek477) [inline package.json:25]: needs to be changed after this [PR](https://github.com/bitfinexcom/bfx-hf-algo/pull/194) is merged.

**Review by tarcisiozf — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (vigan-abd) [reply package.json:22]:
```suggestion
    "bfx-api-node-core": "^1.5.6",
```

File: package.json (hunk 2 above)
Comment (vigan-abd) [reply package.json:25]:
```suggestion
    "bfx-hf-algo": "git+https://github.com/bitfinexcom/bfx-hf-algo.git#v4.10.0",
```

**Review by vigan-abd — APPROVED** [review]

**Pull request merged**